src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/FixedActiveThreadsExecutor.kt

```diff
@@ -25,11 +25,6 @@ import java.util.concurrent.locks.*
  * possible, so that they should not be parked and unparked between invocations.
  */
 internal class FixedActiveThreadsExecutor(private val nThreads: Int, runnerHash: Int) : Closeable {
```

Comment:
Please make a separate PR with the executor fix

Reply:
Here it is: https://github.com/JetBrains/lincheck/pull/202/files